includes/Core/Assets/Assets.php

```diff
@@ -373,12 +366,17 @@ private function get_assets() {
 				'googlesitekit_admin',
 				array(
 					'src'          => $base_url . 'js/googlesitekit-admin.js',
-					'dependencies' => array( 'wp-i18n' ),
+					'dependencies' => array(),
 					'execution'    => 'defer',
 					'before_print' => function( $handle ) {
 						wp_add_inline_script(
 							$handle,
-							'window._googlesitekitBase = ' . wp_json_encode( $this->get_inline_base_data() ),
+							implode(
+								"\n",
+								array(
+									'window._googlesitekitBase = ' . wp_json_encode( $this->get_inline_base_data() ),
+								)
+							),
```

Comment:
Why this change? That looks a bit unnecessary to me :)

Reply:
Ah, I think this was combined with a few others at some point that were later (re)moved. I'll revert this portion 👍

Reply:
Fixed in b7b8cad596c2d2409bd27a29f6fb3a99a0b6a2b2